### PR TITLE
Ontime from db

### PR DESCRIPTION
**Pull request by maxnoe**
Implement database util function to get the correct ontime from our database.

See https://github.com/dneise/about_fact_ontime/blob/master/How%20to%20calculate%20FACTs%20ontime%20from%20auxiliary%20files.ipynb

### CODE REVIEW

File: fact/factdb/utils.py [hunk 1]
```diff
@@ -2,10 +2,57 @@
...
+        RunInfo.fnight.alias('night'),
+        RunInfo.frunid.alias('run_id'),
+        RunInfo.frunstart.alias('start'),
+        RunInfo.frunstart.alias('stop'),
+        duration.alias('duration'),
+        RunInfo.feffectiveon.alias('relative_ontime'),
+    )
+
+    if start is not None:
+        start = night_integer(start) if not isinstance(start, int) else start
```
Comment (kbruegge) [inline fact/factdb/utils.py:51]: why do you have to perform that check here? shouldnt this always be an int?

**Review by kbruegge — APPROVED** [review]

File: fact/factdb/utils.py (hunk 1 above)
Comment (maxnoe) [reply fact/factdb/utils.py:51]: this gives the user the ability to also give an `datetime.date` object for start and end.

Comment (maxnoe) [reply fact/factdb/utils.py:51]: I will update the docstring

Comment (maxnoe) [reply fact/factdb/utils.py:51]: done

**Pull request merged**